src/modules/getScore.js

```diff
@@ -0,0 +1,13 @@
+import showScores from './action';
+
+const ID = 'Neves7';
+
+const getScore = async () => {
+  const response = await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${ID}/scores`);
+  const expect = await response.json();
+  const scoreArr = expect.result;
+  console.log(scoreArr);
+  showScores(scoreArr);
+};
+
+export default getScore();
```

Comment:
[REQUIRED] Kindly fix your code. The newly created score doesn't appear even when I click the refresh button. They only appear when the page is refreshed.